packages/framework/collabspaces/src/collabSpaces.ts

```diff
@@ -511,8 +546,23 @@ export class CollabSpacesRuntime
 		return this.matrixInternal;
 	}
 
+	private updateReverseMap(row: number, col: number) {
+		// the first row and column are used to track the internal IDs
+		const info = this.getCellInfo(row, col);
```

Comment:
Shouldn't this fail? I.e. we are calling something like getCellInfo(5, -1).
I'd suggest calling directly matrix.getCell (and thus not do -1 in the caller of this API), and only once you got everything, call this.reverseMap.addCellToMap() and do -1 logic right before this call